Slighly modified fix pulled up to KDE tree, together with similar fix
to kscd-script. (jdolecek 1999/11/15)

$NetBSD: patch-ae,v 1.2 1999/11/15 23:01:52 jdolecek Exp $
--- kscd/config.h.std.orig	Wed Nov 10 22:16:15 1999
+++ kscd/config.h.std	Wed Nov 10 22:18:53 1999
@@ -123,13 +123,13 @@
 
 #if defined(__FreeBSD__) || defined(__NetBSD__)
 
+#ifdef __FreeBSD__
 /* Is this the most commonly used cd device on FreeBSD ? */
-
 #define DEFAULT_CD_DEVICE       "/dev/rmatcd0c"
 
-/*
+#else /* __NetBSD__ */
 #define DEFAULT_CD_DEVICE       "/dev/rcd0d"
-*/
+#endif /* __FreeBSD__ */
 
 /* Please have a look at void KSCD::playClicked() in kscd.cpp. You will find there
    the following code:
